axelor-admin/src/main/java/com/axelor/apps/base/exceptions/IExceptionMessages.java

```diff
@@ -17,6 +17,7 @@
  */
 package com.axelor.apps.base.exceptions;
 
+@Deprecated
```

Comment:
@alo-axelor I think it would be better to also add the javadoc tag `@deprecated` to say which class should now be used. Same for all the other classes you deprecated.

Reply:
Good idea, I will do this.